x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -37,8 +38,12 @@ import {
 } from '../../../../common/analytics';
 import { LoadingPanel } from '../loading_panel';
 import { getColumnData } from './column_data';
+import { useKibanaContext } from '../../../../../contexts/kibana';
+import { newJobCapsService } from '../../../../../services/new_job_capabilities_service';
+import { getIndexPatternIdFromName } from '../../../../../../application/util/index_utils';
 
 const defaultPanelWidth = 500;
+const NO_KEYWORD_FIELDTYPES = ['boolean', 'integer'];
```

Comment:
should be `const NO_KEYWORD_FIELDTYPES = [ES_FIELD_TYPES.BOOLEAN, ES_FIELD_TYPES.INTEGER];`

Reply:
Changed in 0a79e2a39475beff24b51a40e78f6d6d3e806cca